setup.py

```diff
@@ -60,6 +60,7 @@
     "lightgbm": ["lightgbm>=2,<4"],
     "pytorch": [
         "torch==2.1.2",
+        "tqdm",
```

Comment:
tqdm is needed in PytorchModel, so let's declare it explicitly even though transformers requires it already.